components/oku-circ.vue

```diff
@@ -49,5 +61,130 @@ export default {
 }
 </script>
 
-<style>
+<style lang="scss">
+
+  @import '~susy';
+  @import '~font-awesome/css/font-awesome.min.css';
```

Comment:
Once Nuxt `v1.0` is released, we should be able to switch over to the [Font Awesome module](https://github.com/nuxt-community/modules/tree/master/modules/font-awesome)